### ISSUE TITLE
Restify integration

### ISSUE THREAD

**Issue by calebmer** [ethanresnick/json-api#42]
The [Restify](http://restify.com) framework shares syntax with Express with a few minor variations. Is Restify currently supported, and if not what would be required to ensure integration?

**Comment by ethanresnick** [comment]
Everything I said here about [integrating with Koa](https://github.com/ethanresnick/json-api/issues/37) should apply to restify too. Bottom line: it should be incredibly simple, and there are some instructions in that other issue, but I don't have time to tackle it personally right now. A PR would be welcome, though.

**Comment by NullVoxPopuli** [comment]
@calebmer did you manage to get this working?

**Comment by calebmer** [comment]
No, this was a blocker to me using this library/framework.

**Comment by NullVoxPopuli** [comment]
:-(

**Comment by NullVoxPopuli** [comment]
did you find another json api library?

**Comment by calebmer** [comment]
Nah, I’m using GraphQL now and I’m so happy I made that decision :blush:
